pyproject.toml

```diff
@@ -33,7 +33,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "<3.12,>=3.7.1"
 requests = "^2.25.1"
-singer-sdk = ">=0.28,<0.34"
+singer-sdk = ">=0.31,<0.34"
```

Comment:
This is handled automatically by Depandabot PRs here we should probably just leave this be as I don't see a big value in bumping up the lower end of the singer-sdk range right now (Generally Poetry should just pull the latest SingerSDK version so maybe there's an argument for not having a range but that's pretty seperate

https://github.com/MeltanoLabs/target-postgres/pull/199

Reply:
My logic was that 0.31 is the version that merged https://github.com/meltano/sdk/pull/1864 which was mentioned as a fix for #124. But if Dependabot is handling it, that makes sense.